sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -703,6 +704,30 @@ object StreamExecution {
     "py4j.protocol.Py4JJavaError: An error occurred while calling" +
       s"((.|\\r\\n|\\r|\\n)*)(${IO_EXCEPTION_NAMES.mkString("|")})").r
 
+  private var columnFamilySchemas:
+    Map[UUID, Map[UUID, Map[String, StateStoreColFamilySchema]]] = Map.empty
```

Comment:
I think we need to use a concurrent hashMap or use a mutex to make access to this map thread safe

Reply:
maybe we can just keep it simple and keep a map of `queryId -> ColFamilySchema` and update it on each new run and remove the entry on query stop ?